stdlib/ballerina-http/src/main/ballerina/http/http_load_balancer.bal

```diff
@@ -15,280 +15,254 @@
+    }
+    public function getResponse (HttpFuture httpFuture) returns Response | HttpConnectorError;
+
+    documentation {
+        The hasPromise implementation of the LoadBalancer Connector.
+
+        P{{httpFuture}} The Future which relates to previous async invocation
+        R{{}} Returns true if the push promise exists
+    }
+    public function hasPromise (HttpFuture httpFuture) returns boolean;
+
+    documentation {
+        The getNextPromise implementation of the LoadBalancer Connector.
+
+        P{{httpFuture}} The Future which relates to previous async invocation
+        R{{}} The HTTP Push Promise message or an `HttpConnectorError` if failed to fulfill the request
+    }
+    public function getNextPromise (HttpFuture httpFuture) returns PushPromise | HttpConnectorError;
+
+    documentation {
+        The getPromisedResponse implementation of the LoadBalancer Connector.
+
+        P{{promise}} The related Push Promise message
+        R{{}} The HTTP Push Response message or an `HttpConnectorError` if failed to fulfill the request
+    }
+    public function getPromisedResponse (PushPromise promise) returns Response | HttpConnectorError;
+
+    documentation {
+        The rejectPromise implementation of the LoadBalancer Connector.
+
+        P{{promise}} The Push Promise to be rejected
+    }
     public function rejectPromise (PushPromise promise);
 
 };
 
-@Description {value:"Represents an error occurred in an action of the Load Balance connector."}
-@Field {value:"message: An error message explaining about the error."}
-@Field {value:"cause: The error that caused HttpConnectorError to get thrown."}
-@Field {value:"stackTrace: Represents the invocation stack when LoadBalanceConnectorError is thrown."}
-@Field {value:"statusCode: HTTP status code of the LoadBalanceConnectorError."}
-@Field {value:"httpConnectorError: Array of HttpConnectorError error occurred at each endpoint."}
+documentation {
+    Represents an error occurred in an action of the Load Balance connector.
+
+    F{{message}} An error message explaining about the error
+    F{{cause}} The error that caused HttpConnectorError to get thrown
+    F{{statusCode}} HTTP status code of the LoadBalanceConnectorError
+    F{{httpConnectorError}} Array of HttpConnectorError error occurred at each endpoint
+}
 public type LoadBalanceConnectorError {
     string message,
     error? cause,
     int statusCode,
     HttpConnectorError[] httpConnectorError,
 };
 
-
-@Description {value:"The POST action implementation of the LoadBalancer Connector."}
-@Param {value:"path: Resource path"}
-@Param {value:"request: A Request struct"}
-@Return {value:"The Response struct"}
-@Return {value:"Error occurred during the action invocation, if any"}
-public function LoadBalancer::post (string path, Request? request = ()) returns (Response | HttpConnectorError) {
+public function LoadBalancer::post (string path, Request? request = ()) returns Response | HttpConnectorError {
```

Comment:
Can you format these functions as well? 
- Remove the space between the function name and the parenthesis
- Remove spaces in the union type

Reply:
There are multiple places where we have spaces between union types. Will make the changes?